lib/galaxy/files/__init__.py

```diff
@@ -13,6 +13,7 @@
 from galaxy import exceptions
 from galaxy.files.sources import (
     BaseFilesSource,
+    FilesSourceProperties,
     PluginKind,
```

Comment:
Just für my understanding (still have to learn a lot regarding typing), should this import be hidden behind a check for `typing.TYPE_CHECKING`? And then use as `"FilesSourceProperties"`?

Reply:
Some discussion here: https://github.com/galaxyproject/galaxy/pull/17497#discussion_r1494917419